sample/helloworld/test.yaml

```diff
@@ -1,5 +1,5 @@
 test:
-    syspage: helloworld
+    syspage: main
```

Comment:
maybe we could change it to `exec` or something like that, as we still overuse the `syspage` naming (see description of this PR :P)

Reply:
Done. I also removed some part of building code related to that "syspage" naming.